src/core_modules/capture-core/components/TemplateSelector/hooks/useProgramStageTemplates.ts

```diff
@@ -0,0 +1,30 @@
+import { useFeature, FEATURES } from 'capture-core-utils';
+import { useApiDataQuery } from 'capture-core/utils/reactQueryHelpers';
+import type { WorkingListTemplate } from '../workingListsBase.types';
+
+type DataResponse = {
+    programStageWorkingLists?: WorkingListTemplate[];
+};
+
+export const useProgramStageTemplates = (programId: string | undefined) => {
+    const supportsStoreProgramStageWorkingList = useFeature(FEATURES.storeProgramStageWorkingList);
+    const { error, isLoading, data } = useApiDataQuery<DataResponse>(
+        ['programStageWorkingLists', programId],
+        {
+            resource: 'programStageWorkingLists',
+            params: {
+                filter: programId ? `program.id:eq:${programId}` : '',
```

Comment:
Due to the `enabled` flag on line 21, you do not need to do this check. It will always be a string. Change to just `filter: program.id:eq:${programId}`